src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static final String HORIZONTAL_LINE = "____________________________________________";
+    private static final String INDENT = "    ";
+    private static final int MAX_TASK_LIMIT = 100;
+    private static final int START_OF_STRING = 0;
+
+    private static Task[] tasks;
+    private static String userInput;
+    private static String todoTask;
+    private static String deadlineDescription;
+    private static String by;
+    private static String eventDescription;
+    private static String at;
+    private static int inputNum;
+    private static int doneTaskNum;
+
+    private static int dividerPosition;
+    private static int charAfterDividerPosition;
+
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+
+
+    private static void printLogo() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+
+    private static void printGreeting(String s, String s2) {
```

Comment:
Recommend to add comments for functions to describe what the function does as well as its parameters and return value (if applicable)